src/main/java/io/adobe/cloudmanager/model/PipelineExecutionStepState.java

```diff
@@ -35,6 +35,8 @@
 @EqualsAndHashCode
 public class PipelineExecutionStepState extends io.adobe.cloudmanager.swagger.model.PipelineExecutionStepState {
 
+  private static final long serialVersionUID = 1L;
```

Comment:
I think you need to mark the `client` as transient, because that is not serializable! Then it cannot be final though. I think just exposing the delegate is way easier...